File: Java/nastyhacks.java

// https://open.kattis.com/problems/nastyhacks
import java.util.Scanner;

public class nastyhacks {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cases = sc.nextInt();
        while (cases-- > 0) {
            int noAdvertise = sc.nextInt();
            int ifAdvertise = sc.nextInt();
            int cost = sc.nextInt();
            System.out.println(noAdvertise == ifAdvertise - cost ? "does not matter"
                    : noAdvertise > ifAdvertise - cost ? "do not advertise" : "advertise");
        }
        sc.close();
    }
}

File: C#/nastyhacks.cs
using System;

public class nastyhacks {
    public static void Main (string[] args) {
        int cases = int.Parse (Console.ReadLine ());
        while (cases-- > 0) {
            string[] line = Console.ReadLine ().Split (' ');
            int noads = int.Parse (line[0]);
            int yesads = int.Parse (line[1]);
            int adcost = int.Parse (line[2]);
            Console.WriteLine (yesads - adcost > noads ? "advertise" : yesads - adcost == noads ? "does not matter" : "do not advertise");
        }
    }
}